Bank Mizrahi - Tefahot.py

```diff
@@ -0,0 +1,46 @@
+def test_bank_20(account_number: str, branch_number: str):
+    # Runs a testing function
+    initial_tests(account_number, branch_number)
```

Comment:
@TzviShteren tests should be only while developing, it should be on different file or at least not runnable each time we run the function.